### ISSUE TITLE
KDE Panel Rounded Korners

### ISSUE THREAD

**Issue by TechXero** [vinceliuice/Layan-kde#41]
Would be great if you brought back the KDE Panel'r rounded Corners.. They were there in older versions now they gone.. 

Thanks for the hard work ;)

**Comment by TechXero** [comment]
Bump ?

**Comment by vinceliuice** [comment]
![Screenshot_20220908_122730](https://user-images.githubusercontent.com/7604295/189034313-18b607ff-76e9-425e-b67a-2bff7df0460a.png)

**Comment by TechXero** [comment]
Well installing from AUR Git version gives square corners... What am I doing wrong? Arch + KDE 5.25.5...

**Comment by vinceliuice** [comment]
Just install it from github, AUR Git version is not I managed

**Comment by TechXero** [comment]
Git comes from github... [check it](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=layan-kde-git)

Also I can't coz I need it as a package because I use it as default in my Distro [XeroLinux](https://xerolinux.xyz)

**Comment by Rayrsn** [comment]
what's the output of this command?
```bash
pacman -Qs layan
```

**Comment by Rayrsn** [comment]
Your version is up to date with the git version, i also don't have any problems with the corners

**Comment by TechXero** [comment]
I don't understand why it's not working.. As you can see from my screenshot corners of Panel are squared off.. Also transparency very low.. Weird stuff.. Nothing unusual has been done for it to break xD

**Comment by TechXero** [comment]
> is your compositor running?

Of course it is else it would be totally black..
![image](https://i.imgur.com/pjz8UqF.png)

**Comment by TechXero** [comment]
Gotta figure out what's causing this. Would love to use the KDE panel but as it is I went back to Latte-dock git version...

**Comment by Rayrsn** [comment]
Can you upload it again? the file is removed

**Comment by TechXero** [comment]
> Can you upload it again? the file is removed

Yeah done !
https://mega.nz/file/4CZRxawQ#ivZuX3z05kWMQ3yTphN53tdhedUUVpZGNPLdhw5oJ50

**Comment by Rayrsn** [comment]
I cloned the repo and ran the install script and it got fixed but when i install the aur version it doesn't look right.
i haven't figured out what might be causing the problem

**Comment by TechXero** [comment]
> I cloned the repo and ran the install script and it got fixed but when i install the aur version it doesn't look right.
> 
> i haven't figured out what might be causing the problem

I think it's coz it's in /usr/share not home directory.. Could be wrong

**Comment by Rayrsn** [comment]
yeah i thought that too, but i installed it in the `/usr/share` directory using the original script and it was fine

**Comment by TechXero** [comment]
> yeah i thought that too, but i installed it in the `/usr/share` directory using the original script and it was fine

Hmm.. We uncovered an issue with AUR's PKGBUILD I guess.. Maybe not copying stuff to the right place...

**Comment by TechXero** [comment]
> yeah i thought that too, but i installed it in the `/usr/share` directory using the original script and it was fine

No solution yet ?

**Comment by Rayrsn** [comment]
Nope nothing yet

**Comment by TechXero** [comment]
> Nope nothing yet

I guess gotta be patient.. Thanks for hard work ;)
